### ISSUE TITLE
Mockup designs for national party account data

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#6112]
**What we're after:**
Based on what's learned from the design studio and business requirements, need to mockup designs to make national party account data accessible.

**Related tickets:**
- https://github.com/fecgov/fec-cms/issues/6111
- https://github.com/fecgov/fec-cms/issues/6088
- 

**Completion criteria:**
- [x] Initial designs are created and reviewed by PMs and other required groups

**Comment by JonellaCulmer** [comment]
@PaulClark2 @bmathesonFEC @patphongs @djgarr @fecjjeng 
Here are the mockups for the national party data features. Please provide your feedback when you are able. 

### National party account raising datatable
Please pay special attention to areas in pink. Would need to determine if the placement and language used is what we want. 
![Screenshot 2024-02-29 at 3 45 52 PM](https://github.com/fecgov/fec-cms/assets/31663028/0d152f6f-4025-4929-835f-49856631e488)

### National party account spending datatable
Want to clarify what data we will see in the Description column. Also, please pay special attention to areas in pink. Would need to determine if the placement and language used is what we want. 
![Screenshot 2024-02-29 at 3 46 16 PM](https://github.com/fecgov/fec-cms/assets/31663028/ab1d9ff3-4520-447c-b5ce-5080cf1ac2d5)

### Committee profile pages - financial summary
Need to determine if we will break out the individual accounts into separate lines similar to XXX section.
![Screenshot 2024-02-29 at 3 41 13 PM](https://github.com/fecgov/fec-cms/assets/31663028/f48756fb-5d51-468c-acaf-c271e4d709fc)

### Committee profile page - raising tab
Includes summary figures as well as by transaction. Each account type is in a separate section. Mainly because not all national party committees have all three account types. 
![Screenshot 2024-02-29 at 3 41 38 PM](https://github.com/fecgov/fec-cms/assets/31663028/4b1336b0-ecfa-4788-9f7a-6d567c4c64fc)

### Committee profile page - spending tab
Includes summary figures as well as by transaction. 
![Screenshot 2024-02-29 at 3 41 51 PM](https://github.com/fecgov/fec-cms/assets/31663028/1116d185-da6d-42b9-9d53-d21234a5bb16)

### Browse data page - raising
Needs description language to be drafted.
![Screenshot 2024-02-29 at 3 40 18 PM](https://github.com/fecgov/fec-cms/assets/31663028/27194591-f2f8-418c-b0cb-48095f749ac9)

### Browse data page - spending
Needs description language to be drafted.
![Screenshot 2024-02-29 at 3 40 32 PM](https://github.com/fecgov/fec-cms/assets/31663028/0e36c287-f72a-4a41-908c-ea0584cbd21c)

**Comment by djgarr** [comment]
@JonellaCulmer On the first example, could "Receipt type" be changed to "Party account" or would that throw things off?

Other than that, everything looks good

**Comment by bmathesonFEC** [comment]
Awesome, @JonellaCulmer! These look great. Here are some changes that I think will make the data more clear for the user:

### National party account raising datatable
-	For the title, how about this wording to mirror the existing receipts table: **National party account receipts** 
-	“Source” sounds good.
-	I think the “receipt type” column should be titled “account” or “party account” (like @djgarr suggested) and display the account the receipt is received for.  As a result, the field should only display the account: “Headquarters,” “Recount,” or “Convention”. Based on these mock-ups, it looks like we’re getting the transaction code description instead of the shorter account type.
-	I would expect the “National Party Account Type” filter to only show three options that correspond to the information presented in the “Account” column: “Headquarters,” “Recount,” and “Convention”

### National party account spending datatable
-	Similar comments for the disbursements table as the receipts table
-	For the title, how about this to mirror the existing receipts table: **National party account disbursements** 
-	I think the “disbursement type” column should be titled “account” or “party account” and display the account the disbursement is spent from.  As a result, the field should only display the account: “Headquarters,” “Recount,” or “Convention”.
-	I would expect the “National Party Account Type” filter to only show three options that correspond to the information presented in the “Account” column: “Headquarters,” “Recount,” and “Convention.” Based on these mock-ups, it looks like we’re getting the transaction code description instead of the shorter account type.
-	Everything added here for “description” looks good to me

### Committee profile pages - financial summary
Still thinking about this one. Will follow-up with feedback.

### Committee profile page - raising tab
Still thinking about this one. Will follow-up with feedback.

### Committee profile page - spending tab
Still thinking about this one. Will follow-up with feedback.

### Browse data page - raising
How about this for the descriptive paragraph:

> Search contributions to see who is donating to [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) established to defray certain expenses incurred with respect to presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings. Search for contributors by name, location, employer and occupation, and party account.

(used language from the page linked in the proposed description: https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/)

### Browse data page - spending
How about this for the descriptive paragraph:

> Search disbursements made from [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) established to defray certain expenses incurred with respect to presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings. Search by recipient, purpose, amount, date, and party account.

(used language fromthe page linked in the proposed description: https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/)

**Comment by JonellaCulmer** [comment]
@hcaofec @fecjjeng Is there a column in the database that's just the account type or do we only have transaction codes? This is response to Brad's comments:

> I think the “receipt type” column should be titled “account” or “party account” (like @djgarr suggested) and display the account the receipt is received for. As a result, the field should only display the account: “Headquarters,” “Recount,” or “Convention”. Based on these mock-ups, it looks like we’re getting the transaction code description instead of the shorter account type.

> I would expect the “National Party Account Type” filter to only show three options that correspond to the information presented in the “Account” column: “Headquarters,” “Recount,” and “Convention”

> I think the “disbursement type” column should be titled “account” or “party account” and display the account the disbursement is spent from. As a result, the field should only display the account: “Headquarters,” “Recount,” or “Convention”.

> I would expect the “National Party Account Type” filter to only show three options that correspond to the information presented in the “Account” column: “Headquarters,” “Recount,” and “Convention.” Based on these mock-ups, it looks like we’re getting the transaction code description instead of the shorter account type.

**Comment by JonellaCulmer** [comment]
@bmathesonFEC @djgarr I've made most the changes you requested except for the ones I reached out to Helen and Jean about. 

For the language you drafted, is there any way we can tighten that up a bit? I proposed the following but can revert back to what you proposed if you feel strongly about it. 

Receipts
Search contributions to see who is donating to [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) ~~established to defray certain expenses incurred with respect to~~ `including` presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings. Search for contributors by name, location, employer and occupation, and party account.

Disbursements
Search disbursements made from [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) ~~established to defray certain expenses incurred with respect to~~ `including` presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings. Search by recipient, purpose, amount, date, and party account.

**Comment by bmathesonFEC** [comment]
Totally agree that my version was way too long. I added one word to round out that sentence. Rest of the content team might have suggestions (@djgarr, @kathycarothers, @dorothyyeager, @ryanfec, does Isaac have a github account?)

**Receipts**
Search contributions to see who is donating to [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) ~~established to defray certain expenses incurred with respect to~~ **including** presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings **accounts**. Search for contributors by name, location, employer and occupation, and party account.

**Disbursements**
Search disbursements made from [national party committee accounts](https://www.fec.gov/help-candidates-and-committees/registering-political-party/national-party-accounts-certain-expenses/) ~~established to defray certain expenses incurred with respect to~~ **including** presidential nominating conventions, national party headquarters buildings, and election recounts and contests and other legal proceedings **accounts**. Search by recipient, purpose, amount, date, and party account.

**Comment by kathycarothers** [comment]
For the last sentence, we should remove the comma before the and - date and party account.

**Comment by JonellaCulmer** [comment]
@PaulClark2 @bmathesonFEC Per our discussions, here are updated mocks for the committee profile pages. Need to know which datatables we'll be linking to from the "Other" category in the raising and spending sections of the financial summary.


### Financial summary - raising
![Image](https://github.com/fecgov/fec-cms/assets/31663028/21c5675c-84f0-4601-b440-1279fc89ac6b)

### Financial summary - spending
![Image](https://github.com/fecgov/fec-cms/assets/31663028/5e83250c-c50d-4b05-b45b-dd4d05bdf407)

### Raising tab
![Image](https://github.com/fecgov/fec-cms/assets/31663028/b6c3579f-ebf2-401d-ac77-e7be1cb4f9f0)

### Spending tab
![Image](https://github.com/fecgov/fec-cms/assets/31663028/8cc2be67-ed49-4bb1-80d3-338dbddd98f1)

**Comment by bmathesonFEC** [comment]
@jonella, awesome! Here's some feedback. I've offered these to Debbie too. I'll relay notes if she has any comments.

**Financial summary raising and spending**: In my opinion, "other" can link to the same filtered data table that "Other Receitps" and "Other Disbursements" links to. Since there's no way to filter-out transactions that are attributed to these party accounts, those transactions would be included, but I think that's the best option we have available to us.

**Raising Tab:**
delete this text from the national party account section

"see the financial summary for a breakdown of each type of receipt"
"newly filed summary data may not appear for up to 48 hours"

**Spending Tab:**
delete this text from the national party account section

"see the financial summary for a breakdown of each type of receipt"
"newly filed summary data may not appear for up to 48 hours"

**Comment by JonellaCulmer** [comment]
@brad Thanks for the quick feedback!

> Financial summary raising and spending: In my opinion, "other" can link to the same filtered data table that "Other Receipts" and "Other Disbursements" links to. Since there's no way to filter out transactions that are attributed to these party accounts, those transactions would be included, but I think that's the best option we have available to us.

Okay, no problem. That was my assumption. Thanks for confirming!

> Raising Tab:
> delete this text from the national party account section
> 
> "see the financial summary for a breakdown of each type of receipt"
> "newly filed summary data may not appear for up to 48 hours"

I can delete both, or should I just delete the second line and edit the first line to "...each type of account"?

> Spending Tab:
> delete this text from the national party account section
> 
> "see the financial summary for a breakdown of each type of receipt"
> "newly filed summary data may not appear for up to 48 hours"

Same question ^

**Comment by bmathesonFEC** [comment]
Yep, "See the financial summary for a breakdown of each type of account" looks good.

**Comment by PaulClark2** [comment]
This all looks good to me. Thank you both for keeping this moving.

**Comment by JonellaCulmer** [comment]
Closing in favor of implementation tickets:
- Database: 
  - https://github.com/fecgov/openFEC/issues/5733
- Front-end:
  - https://github.com/fecgov/fec-cms/issues/6179
  - https://github.com/fecgov/fec-cms/issues/6180
  - https://github.com/fecgov/fec-cms/issues/6181
  - https://github.com/fecgov/fec-cms/issues/6182
  - https://github.com/fecgov/fec-cms/issues/6183
